consensus/src/aggregation/types.rs

```diff
@@ -266,10 +258,8 @@ impl<V: Variant, D: Digest> Read for Certificate<V, D> {
     }
 }
 
-impl<V: Variant, D: Digest> EncodeSize for Certificate<V, D> {
-    fn encode_size(&self) -> usize {
-        self.item.encode_size() + self.signature.encode_size()
-    }
+impl<V: Variant, D: Digest> FixedSize for Certificate<V, D> {
```

Comment:
To store `Certificate` in `Ordinal`, we need this to be fixed size like `threshold_simplex::Seed`.